src/content/community/grants/index.md

```diff
@@ -28,17 +28,16 @@ These programs support the broad Ethereum ecosystem by offering grants to a wide
 These projects have created their own grants for projects aimed at developing and experimenting with their own technology.
 
 - [Aave Grants Program](https://aavegrants.org/) – _[Aave](https://aave.com/) grants DAO_
-- [Balancer](https://balancergrants.notion.site/Balancer-Community-Grants-23e562c5bc4347cd8304637bff0058e6) – _[Balancer](https://balancer.fi/) ecosystem fund_
+- [Balancer](https://quark-ceres-740.notion.site/Balancer-Grants-938f1b979810427f8d903a904315da41) – _[Balancer](https://balancer.fi/) ecosystem fund_
 - [Chainlink Grants Program](https://chain.link/community/grants) - _[Chainlink](https://chain.link/) community grants_
-- [Compound Grants Program](https://compoundgrants.org/) – _[Compound](https://compound.finance/) finance ecosystem_
+- [Compound Grants Program](https://twitter.com/compoundgrants/status/1615562418297659393?s=20) – _[Compound](https://compound.finance/) finance ecosystem_
```

Comment:
I'm not really happy about linking to a Twitter post here; is there a better way?

Reply:
The problem is that there are multiple grant programs and I have not found another link / page that would contain all of them.

I can use https://linktr.ee/ or some similar service and just paste the links there. Will it be better?

Reply:
So looks like they use Questbook to manage this, and post links to new grants rounds on Twitter. I don't think we can maintain this, so I would just remove it entirely.